examples/CloudRetro/README.md

```diff
@@ -0,0 +1,509 @@
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: worker
+  template:
+    metadata:
+      labels:
+        app: worker
+    spec:
+      containers:
+      - name: worker
+        image: valniae/snekyrepo:cloudimage4
+        volumeMounts:
+        - name: cloudretro-confmap-vol
+          mountPath: "/usr/local/share/cloud-game/configs/"
+          readOnly: true
+        command: ["worker"]
+        args: ["--v=5"]
+        ports:
+        - containerPort: 9000
+          containerPort: 8443
+      volumes:
+      - name: cloudretro-confmap-vol
+        configMap:
+          name: cloudretro-config
+EOF
+```
+
+This should leave us with working pods of workers:
+
+```console
+kubectl get pods -n cloudretro
+
+NAME                                      READY   STATUS    RESTARTS   AGE
+coordinator-deployment-546ff547cb-qpnsx   1/1     Running   0          18h
+worker-deployment-c4d4d8f8f-2dwdn         1/1     Running   0          5m25s
+worker-deployment-c4d4d8f8f-f8b4h         1/1     Running   0          5m26s
+```
+
+Now we will need to make a service for the workers HTTP server just like for the coordinator. For this, we will be using the port 9000. Again, if necessary, you can configure the NodePort as well.
+(note: we are using LoadBalancer, because we will need to expose that for the public)
+(won`t a LB kill the purpose of a ping-check though?)
+
+```console
+kubectl apply -f - <<EOF
+apiVersion: v1
+kind: Service
+metadata:
+  name: worker-lb-svc
+  namespace: cloudretro
+spec:
+  selector:
+    app: worker
+  ports:
+    - port: 9000
+      targetPort: 9000
+      #nodePort: 30002
+  type: LoadBalancer
+EOF
+```
+
+Another k8s service will be needed for the UDP service, although this time we will use a ClusterIP. Because of STUNner, we will be using the in-cluster network for that.
+For the UDP port, you should be using the one you`ve used at the worker deployment. NodePort is not needed here, because we are not planning to expose this service.
+
+```console
+kubectl apply -f - <<EOF
+apiVersion: v1
+kind: Service
+metadata:
+  name: worker-ci-udp-svc
+  namespace: cloudretro
+spec:
+  selector:
+    app: worker
+  ports:
+    - protocol: UDP
+      port: 8443
+      targetPort: 8443
+  type: ClusterIP
+EOF
+```
+
+Now, checking up on services should be looking like this:
+
+```console
+kubectl get services -n cloudretro
+
+NAME                 TYPE           CLUSTER-IP            EXTERNAL-IP             PORT(S)          AGE
+coordinator-lb-svc   LoadBalancer   <YOUR_C_CLUSTER_IP>   <YOUR_C_EXTERNAL_IP>    8000:30618/TCP   18h
+worker-ci-udp-svc    ClusterIP      10.104.129.8          <none>                  8443/UDP         3m2s
+worker-lb-svc        LoadBalancer   <YOUR_W_CLUSTER_IP>   <YOUR_W_EXTERNAL_IP>    9000:32681/TCP   3m10s
+```
+
+Note down <YOUR_C_CLUSTER_IP>, <YOUR_C_EXTERNAL_IP>, <YOUR_W_EXTERNAL_IP>. For the next configuration steps, we will need these.
+
+#### CloudRetro config once again
```

Comment:
So far, we have concentrated in setting up cloudretro in Kubernetes. this is not quite interesting here: our goal is to teach STUNner, not cloudretro. please collect all the above into a single yaml, let the users apply that yaml in a single step, and give a high-level decription of what happens here, maybe a bullet list describing the different resources created by the YAML (Deplyoments, Services, whatnot)